cou/apps/base.py

```diff
@@ -243,33 +239,26 @@ def target_channel(self, target: OpenStackRelease) -> str:
         return f"{target.codename}/stable"
 
     @property
-    def current_os_release(self) -> OpenStackRelease:
-        """Current OpenStack Release of the application.
+    def os_release_units(self) -> defaultdict[OpenStackRelease, list[str]]:
```

Comment:
This should not return default dict, but rather just dict. It does not make sense if I call `os_release_units.os_release_units["no-real-unit/99"]` and it works. I think that maybe even tuple of tuple (e.g. `(("unit/0", "1.1"), ("unit/1", "1.2"), )`) will be better.